docs/Early Access - Using Global Profile Configuration.md

```diff
@@ -84,11 +93,11 @@ To begin, define a connection to z/OSMF and initialize your configuration files.
 
    The CLI provides a series of prompts.
 
-2. Respond to the prompts to enter a service name, username, and password for a mainframe service such as z/OSMF. The `--global` option ensures that your credentials are stored securely on your computer by default.
+2. Respond to the prompts to enter a username, and password for a mainframe service such as z/OSMF. The `--global` option ensures that your credentials are stored securely on your computer by default.
```

Comment:
Is it really the --global flag which ensures that credentials are stored securely?

Reply:
@t1m0thyj I'll need your input on this.

Reply:
The `--global` option stores the JSON configuration file in your CLI home directory (default location `~/.zowe`). If omitted, the JSON file would get created in the current working directory. The --global flag is not related to secure credential storage.